code/modules/antagonists/traitor/traitor.dm

```diff
@@ -0,0 +1,421 @@
+
```

Comment:
Random empty space.

Might want to name this file datum_traitor, like TG, to avoid confusion with traitor, the gamemode

Reply:
Righto, I'll name it datum_traitor